src/fmu/dataio/_utils.py

```diff
@@ -149,7 +148,7 @@ def export_file(
             # pa.ipc.IpcWriteOptions(). It is convenient to use feather since it
             # has ready configured defaults and the actual file format is the same
             # (https://arrow.apache.org/docs/python/feather.html)
-            feather.write_feather(obj, dest=filename)
+            feather.write_feather(obj, dest=str(filename))
```

Comment:
@jcrivenaes Could you take a look at this? write_feather wants the input to be a pandas dataframe.

Reply:
Might be a not great annotation from them, docs say a `Table` is fine https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html

Reply:
ohh nice find.